dbcon/joblist/groupconcat.cpp

```diff
@@ -714,7 +719,7 @@ void GroupConcator::outputRow(std::ostringstream& oss, const rowgroup::Row& row)
 
 bool GroupConcator::concatColIsNull(const rowgroup::Row& row)
 {
-  return ranges::any_of(fConcatColumns, [&](uint32_t idx) { return row.isNullValue(idx); });
+  return boost::algorithm::any_of(fConcatColumns, [&](uint32_t idx) { return row.isNullValue(idx); });
```

Comment:
```suggestion
  return std::any_of(fConcatColumns.cbegin(), fConcatColumns.cend(), [&row](uint32_t idx) { return row.isNullValue(idx); });
```

Reply:
Funny fact, I've just suggested the same thing )

Reply:
++